dd-sdk-android-core/src/testFixtures/kotlin/com/datadog/android/api/instrumentation/network/RequestInfoAssert.kt

```diff
@@ -0,0 +1,38 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-Present Datadog, Inc.
+ */
+
+package com.datadog.android.api.instrumentation.network
+
+import org.assertj.core.api.AbstractObjectAssert
+import org.assertj.core.api.Assertions
+
+class RequestInfoAssert private constructor(actual: HttpSpecRequestInfo) :
+    AbstractObjectAssert<RequestInfoAssert, HttpSpecRequestInfo>(actual, RequestInfoAssert::class.java) {
+
+    fun hasUrl(url: String) = apply {
+        Assertions.assertThat(actual.url).isEqualTo(url)
```

Comment:
minor: it is worth to override default error message as well here and below, for the clarity in the logs